app/src/main/java/edu/wisc/ece/pinpoint/utils/LocationChangeDetection.java

```diff
@@ -75,8 +75,6 @@ public Result doWork() {
                                         newLong, Double.parseDouble(preferences.getString("longitude", "")));
                                 Log.d("dist", String.valueOf(distance));
                                 if (newLoc != null && (distance <= 0.5)) {
-                                    
-
```

Comment:
instead of leaving this if statement blank, can you make it the opposite case and put the code below? if (newLoc != null && (distance > 0.5))